docs/microbit_micropython_api.rst

```diff
@@ -105,6 +135,30 @@ Each of these pins are instances of the ``MicroBitPin`` class, which offers the
     # returns boolean
     pin.is_touched()
 
+Except in the case of the pins marked **V2**, which offers the following API::
+    
+    **pin_logo**
+
+    # returns boolean
+    pin_logo.is_touched()
+
+    **pin_speaker**
```

Comment:
we need to add the enable/disable: https://github.com/microbit-foundation/micropython-microbit-v2/issues/17#issuecomment-730320900

Reply:
This has also been implemented 👍